src/utils.ts

```diff
@@ -1422,17 +1422,15 @@ export function compareObjectId(oid1?: ObjectId | null, oid2?: ObjectId | null):
   return oid1.id.compare(oid2.id);
 }
 
-export function getInt(name: string, value: unknown): number {
+export function parseInt(value: unknown): number | null {
```

Comment:
```suggestion
export function parseInteger(value: unknown): number | null {
```

By shadowing a global with a hoisted declaration you would change all the code in the file to use this function instead of the global. Renaming it is a quick fix but ideally a linter would catch this if not tests. (not something I'm suggesting we look into here)

Also, luckily we currently don't have any code using parseInt, I've always used `Number.parseInt` but there's no lint or rule that made me do that, just 🧅